omnigibson/prims/rigid_prim.py

```diff
@@ -175,11 +178,15 @@ def update_meshes(self):
                     mesh.set_rest_offset(m.DEFAULT_REST_OFFSET)
                     self._collision_meshes[mesh_name] = mesh
 
-                    volume, com = get_mesh_volume_and_com(mesh_prim)
+                    is_volume, volume, com = get_mesh_volume_and_com(mesh_prim)
                     vols.append(volume)
                     # We need to translate the center of mass from the mesh's local frame to the link's local frame
                     local_pos, local_orn = mesh.get_local_pose()
                     coms.append(T.quat2mat(local_orn) @ (com * mesh.scale) + local_pos)
+                    # If we're not a valid volume, use bounding box approximation for the underlying collision approx
+                    if not is_volume:
+                        log.warning(f"Got invalid (non-volume) collision mesh: {mesh.name}")
+                        mesh.set_collision_approximation("boundingCube")
```

Comment:
hmm if we don't do this, what would happen?

Reply:
We get the non-compatible GPU mesh error